### PR TITLE
Supporting the write concern option via an environment variable

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- 100% tests passing
- Assigned to @frbattid

### CODE REVIEW

File: src/sth_configuration.js [hunk 1]
```diff
@@ -70,6 +70,16 @@
   module.exports.SERVICE_PATH = ENV.SERVICE_PATH || '/';
   module.exports.POOL_SIZE = (ENV.POOL_SIZE && !isNaN(ENV.POOL_SIZE) && parseInt(ENV.POOL_SIZE) > 0) ?
     parseInt(ENV.POOL_SIZE) : 5;
+  try {
```
Comment (frbattid) [inline src/sth_configuration.js]: This new parameter must be had into account in the `stf_default.conf` file within PR https://github.com/telefonicaid/IoT-STH/pull/45

Comment (frbattid) [inline src/sth_configuration.js]: NTC from this PR point of view :)

**Comment by frbattid** [comment]
LGTM from this PR point of view. Please, take a look on my comment, it may affect other opened PRs (https://github.com/telefonicaid/IoT-STH/pull/45) ;)

**Comment by gtorodelvalle** [comment]
Comment https://github.com/telefonicaid/IoT-STH/pull/45#discussion_r29835022 included in https://github.com/telefonicaid/IoT-STH/pull/45 as suggested by @frbattid ;) Thanks!

**Pull request merged**